lib/spack/docs/environments.rst

```diff
@@ -11,21 +11,21 @@ Environments
 
 An environment is used to group together a set of specs for the
 purpose of building, rebuilding and deploying in a coherent fashion.
-Environments provide a number of advantages over the a la carte
+Environments provide a number of advantages over the *à la carte*
```

Comment:
Hopefully unicode characters are okay. Since we also have localized docs in Japanese, I imagine they are.